src/fsharp/lexhelp.fsi

```diff
@@ -35,7 +35,9 @@ type lexargs =
       applyLineDirectives: bool
 
       /// The degree of nesting of '{..}' in an interpolation fill
-      mutable interpolatedStringNesting: int
+      mutable interpolatedStringNestingCurrent: int
```

Comment:
I might be missing something, but would the introduction of the stack obviate the need for a counter? Might be simpler just to use one thing instead of two.

Reply:
In some sense yes, but the stack degenerates to a stack of two types of symbols, '{' and 'SEP'
The counter folds multiple '{' into an integer.

Reply:
hmm, or we could say, the "current" is the top of the stack?

Reply:
@cartermp you're right. I've refactored the code to remove the explicit stack top and it's cleaner. thanks!